Dockerfile

```diff
@@ -0,0 +1,12 @@
+## Dockerfile
+
+FROM node:alpine
```

Comment:
Shouldn't the base image be a fixed Node.js version? That way, there won't be any breaking changes between two major versions of Node.js and the requirements of the project in the `package.json`. Plus, this does not prevent from one user that has cloned the project one year ago to have a different environment from someone that just cloned the project two minutes ago. This defies the principle of Docker in a way that allows to run the project on one unique configuration (but here, the Node.js version can change anytime in the life of the project when the project is cloned).

Reply:
Thanks. Fixed a node version.

Reply:
`node:14.16.1-alpine3.10` is quite old. I've built successfully with `node:16-alpine3.13`. Anything that would speak against this?